client/public/locales/en/translation.json

```diff
@@ -172,6 +172,8 @@
     "autoSelectTooltip": "Automatically select this answer based on tags associated with the application(s) or archetype.",
     "autoTagTooltip": "Automatically tag this application or archetype with these tags based on this answer to the questionnaire.",
     "assessmentStakeholderHeader": "Select the stakeholder(s) or stakeholder group(s) associated with this assessment.",
+    "issuesEffortTooltip": "This column shows the effort weight for a single issue incident.",
+    "applicationEffortTooltip": "This column shows the total application effort.",
```

Comment:
Well, changing network helped. And I found where you can add the 3rd tooltip if you want to.
Apart from that, LGTM
![Selection_553](https://github.com/konveyor/tackle2-ui/assets/60384172/8662ba4c-2c00-4d25-b5a3-41be35900a06)

Reply:
Done @mguetta1 
![Screenshot from 2024-05-03 13-00-52](https://github.com/konveyor/tackle2-ui/assets/87791967/ae3b4fe3-dcb5-4af5-ace5-3cd842143f54)